src/Compilers/VisualBasic/Portable/Binding/Binder_Expressions.vb

```diff
@@ -1458,7 +1470,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     Return member.AccessKind <> LateBoundAccessKind.Get AndAlso member.AccessKind <> LateBoundAccessKind.Call
 
                 Case Else
-                    Return expression.IsLValue()
+                    Return expression.IsLValue
```

Comment:
>expression.IsLValue [](start = 27, length = 19)

Is this a method? If so, I would appreciate the (superfluous) parenthesis. Their omission in VB often confuses me :)